# 010_BUG349921_toolkit_branding.diff
# allow to use Branding directory in xulrunner configuration
Index: mozilla/toolkit/toolkit-tiers.mk
===================================================================
--- mozilla.orig/toolkit/toolkit-tiers.mk
+++ mozilla/toolkit/toolkit-tiers.mk
@@ -265,8 +265,16 @@ endif
 
 ifdef ENABLE_TESTS
 tier_toolkit_dirs	+= testing/mochitest
 endif
 
 ifdef MOZ_TREE_FREETYPE
 tier_external_dirs	+= modules/freetype2
 endif
+
+ifndef MOZ_PHOENIX
+ifdef MOZ_XULRUNNER
+ifdef MOZ_BRANDING_DIRECTORY
+tier_toolkit_dirs += $(MOZ_BRANDING_DIRECTORY)
+endif
+endif
+endif
